docs/fast_data/single_view_trigger_generator.md

```diff
@@ -44,24 +43,27 @@ Additional details on how to configure the Real-Time Updater to produce Projecti
 
 #### Single View Creator
 
+:::note
+The following steps are only required if you choose to generate `sv-trigger` messages over Projection changes
```

Comment:
how the user can enable it? do we have a documentation section to be linked here?